src/Lykke.Service.ConfirmationCodes/Models/Validation/PhoneValidator.cs

```diff
@@ -0,0 +1,18 @@
+﻿using FluentValidation;
+using JetBrains.Annotations;
+using Lykke.Service.ConfirmationCodes.Client.Models.Request;
+
+namespace Lykke.Service.ConfirmationCodes.Models.Validation
+{
+    [UsedImplicitly]
+    public class PhoneValidator<T> : AbstractValidator<T>
+        where T : SmsConfirmationRequest
+    {
+        public PhoneValidator()
+        {
+            RuleFor(x => x.Phone)
+                .Must(x => !string.IsNullOrWhiteSpace(x) && x.Length >= 8)
```

Comment:
I think 8 is too much
https://stackoverflow.com/questions/14894899/what-is-the-minimum-length-of-a-valid-international-phone-number

Reply:
Hmm - we expect international phone number, so acording to this link it should be 7 digits (shortest internal country phone) + 1 digits (shortest country code) = 8